presto-docs/src/main/sphinx/release/release-0.144.rst

```diff
@@ -7,3 +7,10 @@ General Changes
 
 * Fix already exists check when adding a column to be case-insensitive.
 * Fix correctness issue when complex grouping operations have a partitioned source.
+* Discard output stage json from completion event when it is very long. Add
+  ``event.max-output-stage-size`` config to adjust the threshold.
+
+Hive Changes
+------------
+
+* Add support for creating and inserting into bucketed Hive tables.
```

Comment:
Looks like we didn't even document partitioned_by. Let me do it all together sometime later.